docs/devbook/bump-versions/bump_versions.sh

```diff
@@ -99,14 +146,20 @@ else
   then
     UPDATED_CRATES="\n${UPDATED_CRATES}"
   fi
+  UPDATED_PACKAGE_JSONS="$(echo "$PACKAGE_JSON_FILES" | xargs git diff "$COMMIT_REF" | grep -E "^[\+\-] *\"version\": \"[0-9\.]+\"|name\": " | tr '\n' ' ' | sed -r "s/ *\"name\": \"([a-z@\-]*\/)?([a-z\-]+)\", -  \"version\": \"([0-9\.]+)\", \+  \"version\": \"([0-9\.]+)\", ?/* [js] \2 from \`\3\` to \`\4\`\n/g")"
```

Comment:
We may put some explanation somewhere (here or at the beginning of the script)
It could be just a short example of what we have as input and what we want as output to help understanding what we are doing.

Reply:
:heavy_check_mark: I've added a comment, see few lines above, that contains the transformation that we want to do for those messages.